CardIOSample/CardIOSample/DisplaySVViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  DisplaySVViewController.swift
+//  CardIOSample
+//
+//  Created by Philip Sawyer on 5/26/16.
+//  Copyright © 2016 Philip Sawyer. All rights reserved.
+//
+
+import UIKit
+
+class DisplaySVViewController: UIViewController, CardIOViewDelegate {
+    
+    @IBOutlet weak var scanCardButton: UIButton!
+    @IBOutlet weak var resultsLabel: UILabel!
+    @IBOutlet weak var cancelButton: UIButton!
+    
+    var cardIOView : CardIOView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = "CardIO as SubView"
+        
+        if !CardIOUtilities.canReadCardWithCamera() {
+            self.scanCardButton.hidden = true
+            self.resultsLabel.text = "Camera is not supported"
+        } else {
+            self.resultsLabel.hidden = true
+        }
+        self.cancelButton.hidden = true
+        
+        let rect = CGRect(x: 10, y: 80, width: 400, height: 400)
```

Comment:
If preferring `let` is instinct, run with it. Start with `let`s and weaken to var if needed. Same thinking when thinking about optionals and non-optionals. Start with non, weaken to optional.